test/sql/types/enum/test_enum_to_numbers.test

```diff
@@ -0,0 +1,124 @@
+# name: test/sql/types/enum/test_enum_to_numbers.test
+# description: ENUM types used in table tests
+# group: [enum]
+
+statement ok
+create type enum1 as enum ('1', '2', '3', '4');
```

Comment:
Can we also test with other data types?

Reply:
Enums can only hold varchar values, do you mean compare the enum with other types?

Reply:
These are integers now, right? I imagine similar comparisons could happen with other data types, hugeints, doubles, dates, times?